app/controllers/integrations/amazon_basecamp_controller.rb

```diff
@@ -1,11 +1,7 @@
-require 'json'
-require 'httparty'
-
-class Api::V1::MessagesController < ApplicationController
+class Integrations::AmazonBasecampController < ApplicationController
```

Comment:
this is chatbot for basecamp, so having Basecampe in title of every controller makes little sense. "AwsSnsController" would be better, as it's really only for SNS.